scripts/f5-config.py

```diff
@@ -60,6 +60,9 @@
     r'create ltm monitor http ' + PART + PREFIX_NAME + '_MON_HTTP_KEYSTONE_ADMIN { defaults-from'
     r' http destination *:35357 recv "200 OK" send "HEAD /v3 HTTP/1.1\r\nHost:'
     r' rpc\r\n\r\n" }',
+    r'create ltm monitor http ' + PART + PREFIX_NAME + '_MON_HTTP_MAGNUM { defaults-from'
+    r' http destination *:9511 recv "200 OK" send "HEAD /v1 HTTP/1.1\r\nHost:'
+    r' rpc\r\n\r\n" }'
```

Comment:
Once I added the comman, the monitor appeared just fine:
![screenshot from 2017-04-10 14 14 34](https://cloud.githubusercontent.com/assets/4964964/24878374/82b3c37a-1df8-11e7-9fcb-302a180ed803.png)